medusa/storage/azure_storage.py

```diff
@@ -115,6 +115,7 @@ async def _upload_object(self, data: io.BytesIO, object_key: str, headers: t.Dic
             name=object_key,
             data=data,
             overwrite=True,
+            standard_blob_tier=self.get_storage_class(),
```

Comment:
I'm sorry but this does not work. The Azure library needs to have the instances of `StandardBlobTier` clsss (which are just an enum) here.
So we need to do two things:
- on line 28 import the class (`from azure.storage.blob import BlobProperties, StandardBlobTier`)
- map the config values to the values of the enum and pass the whole enum (down the call stack, the library does call `.value()` on the enum.